wrapper/src/main/java/com/splunk/support/lambda/configuration/SplunkConfiguration.java

```diff
@@ -14,32 +14,28 @@
  * limitations under the License.
  */
 
-package com.splunk.support.lambda;
+package com.splunk.support.lambda.configuration;
 
-import static com.splunk.support.lambda.ExportersInitializer.configureExporters;
-import static com.splunk.support.lambda.PropagatorsInitializer.configurePropagators;
+import static com.splunk.support.lambda.configuration.JaegerThriftSpanExporterFactory.OTEL_EXPORTER_JAEGER_ENDPOINT;
 
-import io.opentelemetry.instrumentation.api.config.Config;
-import io.opentelemetry.instrumentation.api.config.ConfigBuilder;
-import io.opentelemetry.sdk.OpenTelemetrySdk;
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.spi.config.PropertySource;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.logging.ConsoleHandler;
 import java.util.logging.Handler;
 import java.util.logging.Level;
 import java.util.logging.SimpleFormatter;
-import java.util.regex.Pattern;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class Configurator {
+@AutoService(PropertySource.class)
+public class SplunkConfiguration implements PropertySource {
```

Comment:
I'm afraid that this won't work: in otel javaagent `PropertySource`s are loaded in the `javaagent-tooling` module, which is not visible or included by the library instrumentations. There's nothing that consumes theses properties; you'll probably have to go back to setting system properties

Reply:
Shot, I though it's also autoloaded by SDK autoconfigure.

Reply:
OK, done (back with sys props).